pkgs/development/interpreters/python/pypy/2.7/default.nix

```diff
@@ -113,6 +113,7 @@ let
        # will think the wrapper is the original. As long as the wrapper has
        # the same path as the original, this is OK.
        wrapProgram "$out/pypy-c/pypy-c" \
+         --set TIX_LIBRARY ${tix}/lib \
```

Comment:
grepping the PyPy source code gave me the following file that needs to be patched
```
lib-python/2.7/lib-tk/Tix.py:        tixlib = os.environ.get('TIX_LIBRARY')
```